esp/templates/program/modules/surveymanagement/manage.html

```diff
@@ -0,0 +1,176 @@
+{% extends "main.html" %}
+
+{% block title %}{{program.niceName}} Survey Administration{% endblock %}
+
+{% block stylesheets %}
+    {{ block.super }}
+    <link rel="stylesheet" href="/media/styles/forms.css" type="text/css" />
+    <link rel="stylesheet" href="/media/styles/expand_display.css" type="text/css" />
+    <link rel="stylesheet" href="/media/styles/survey_manage.css" type="text/css" />
+{% endblock %}
+
+{% block xtrajs %}
+    {{ block.super }}
+    <script type="text/javascript">
+        var question_types = {{ question_types|safe }};
+    </script>
+{% endblock %}
+
+{% block content %}
+<h1>Survey Management for {{ program.niceName }}</h1>
+
+<p>
+You can use this page to set up surveys for teachers or students of the program. You will first need to create a survey, then you can add questions to it.
+</p>
+
+<p>
+Each step below is a form that you can expand or contract by clicking on the header. If you'd like to import a survey from a previous program, you can do that in Step 1.
+</p>
+
+<div id="program_form">
+    {% ifequal open_section "survey" %}
+    <button class="dsphead active">
+       <b>Step 1: Add Surveys</b> (click to expand/contract)
+    </button>
+
+    <div class="dspcont active">
+    {% else %}
+    <button class="dsphead">
+       <b>Step 1: Surveys</b> (click to expand/contract)
+    </button>
+
+    <div class="dspcont">
+    {% endifequal %}
+    <form method="post" action="/manage/{{ program.url }}/surveys/manage?obj=survey&op=import">
+    <table align="center" cellpadding="0" cellspacing="0" width="100%">
+        <tr><th colspan="2">Import a survey from a previous program</th></tr>
+        {{ import_survey_form }}
+        <tr><td colspan="2" align="center"><input class="fancybutton" type="submit" value="Import Survey" /></td></tr>
+    </table>
+    </form>
+    <br />
+    <form method="post" action="/manage/{{ program.url }}/surveys/manage?obj=survey">
+        {% if survey_form.id %}
+        <input type="hidden" name="survey" value="{{ survey_form.id }}" />
+        {{ survey_form.id }}{% endif %}
+        <table align="center" cellpadding="0" cellspacing="0" width="100%">
+            <tr><th colspan="2">Add/modify a Survey</th></tr>
+            {{ survey_form.as_table }}
+            <tr><td colspan="2" align="center"><input class="fancybutton" type="submit" value="Create/Save Survey" /></td></tr>
```

Comment:
When this button gets pressed, can you auto-update the survey dropdown for step 2? I seem to need to refresh/hit some other things to get it to update rather than just creating a new survey.